pkg/util/limitrange/limitrange.go

```diff
@@ -48,3 +51,67 @@ func addToSummary(summary, item corev1.LimitRangeItem) corev1.LimitRangeItem {
 	summary.MaxLimitRequestRatio = resource.MergeResourceListKeepMin(summary.MaxLimitRequestRatio, item.MaxLimitRequestRatio)
 	return summary
 }
+
+func violateMessage(path *field.Path, isMore bool) string {
+	if isMore {
+		return fmt.Sprintf("the requests of %s exceeds the limits", path.String())
+	}
+	return fmt.Sprintf("the requests of %s are less than the limits", path.String())
```

Comment:
worth having two functions instead.

Also, these messages are not giving any indication which limits this is referring to.